docker-compose.yml

```diff
@@ -1,9 +1,21 @@
 version: '2'
 
 services:
+  mongo:
+    image: mongo:3.5
+    container_name: sedaily-mongo
+
+#    ports:
+#      - "27017:27017"
+  redis:
+    image: redis:4
+    container_name: sedaily-redis
+#    ports:
+#      - "6379:6379"
   express-mongoose-es6-rest-api:
     build:
       context: .
+    container_name: sedaily-rest-api
     volumes:
```

Comment:
We should probably have a volume for the dev environment to link directly to the app

Reply:
There is already a volume attached from host to the container. Is this what you mean? 
```yml
volumes:
      # Mounts the project directory on the host to /app inside the container,
      # allowing you to modify the code without having to rebuild the image.
      - .:/app
      # Just specify a path and let the Engine create a volume.
      # Data present in the base image at the specified mount point will be copied
      # over to the new volume upon volume initialization.
      # node_modules from this new volume will be used and not from your local dev env.
      - /app/node_modules/
```

Reply:
Ah, my mistake, I didn't realize github had closed the section.